storybook/pages/CommunityPermissionsSettingsPanelPage.qml

```diff
@@ -80,6 +82,12 @@ SplitView {
             SplitView.preferredHeight: 150
 
             logsView.logText: logs.logText
+
```

Comment:
You could also add it in `CommunityNewPermissionViewPage`!

Reply:
Sure!